DigitalOffice/Services/UserService/Validators/GetUserByEmailValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+
+namespace LT.DigitalOffice.UserService.Validators
+{
+    public class GetUserByEmailValidator : AbstractValidator<string>
+    {
+        public GetUserByEmailValidator()
+        {
+            RuleFor(email => email)
+                    .NotEmpty()
+                    .WithMessage("Email is empty.")
+                    .MaximumLength(50)
```

Comment:
This is not a valid limitation. Check out the thread in the Talgat thread.

Reply:
Fixed